samples/VSSample.Tests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,20 @@
+using System.Reflection;
```

Comment:
Let's not add AssemblyInfo.cs for this project. I don't believe it is normally included when using the new .NET Standard project system.